### DISCUSSION TITLE
Add web socket intents

### DISCUSSION THREAD

**Discussion by Darkflame72** [stoatchat/discussions#85] [Feature Suggestions]
Add the ability for bot/user account to specify what updates they wish to receive. This will help cut down on traffic sent to bots and the amount of processing that need to do.

Updates such as:
- Status updates
- typing

Are generally not used and are some of the most commonly sent messages. This would function similar to how discord implements it with the ability to specify the wanted intents at the web socket creation. This would not have privilege intents but purely function as an ability for the bot/user to only receive events they want.

This would be an opt-in feature with the default having all messages sent to the client/bot over the websocket

**Comment by oleggtro** [comment]
I think this is probably pretty low prio as long as there are no massive servers
